hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpFilesSubcommand.java

```diff
@@ -318,7 +424,7 @@ Pair<FileSummary, Map<Integer, HederaFile>> collectFiles(
 
         final var nType = new ConcurrentHashMap<VirtualBlobKey.Type, Integer>();
         final var nNullValues = new ConcurrentHashMap<VirtualBlobKey.Type, Integer>();
-        final var nNulLMetadataValues = new AtomicInteger();
+        final var nNullMetadataValues = new AtomicInteger();
```

Comment:
What is the `n` prefix?  It's easy to miss if one doesn't read slowly.

Reply:
`n` == "number" (aka count) -